libs/core/src/components/sage-textarea/sage-textarea.tsx

```diff
@@ -0,0 +1,164 @@
+import { Component, Element, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+import { ChangeEvent } from 'react';
+import { isRequired } from '../../utils/utils';
+
+
+@Component({
+  tag: 'sage-textarea',
+  styleUrl: 'sage-textarea.scss',
+  shadow: true,
+})
+export class SageTextarea {
+  @Element() el: HTMLDivElement;
+
+  /**
+   * A unique identifier for the textarea
+   */
+  @Prop() componentId?: string;
+
+  /**
+   * Indicates whether or not the textarea is disabled
```

Comment:
For the boolean props, I believe we agreed to use "Determines" instead of "Indicates". Not a blocker, since it will depend on Ashley's upcoming guidelines/documentation

Reply:
I was saving this for the completion of the guidelines work. It would likely be handled in a sweeping change